### PR TITLE
Add instruction builder example and links

### PR DESCRIPTION
**Pull request by febo**
Update Rust examples to use the new instruction builders and include links to the JS examples.

### CODE REVIEW

File: token-metadata/program/ProgrammableNFTGuide.md [hunk 1]
```diff
@@ -4,320 +4,262 @@
...
 
 ## Unified instructions
 
-To interact with the new asset class, a new unified set of instruction will be added to Token Metadata. It is important to note that current instructions will continue to work using the existing token standard – they will be required for interacting with `ProgrammableNonFungible` assets. At the same time, the new instructions will support all asset classes so all interaction can happen via an unified interface regardless of the asset class.
+To interact with the new asset class, a new set of instruction will be added to Token Metadata. It is important to note that current instructions will continue to work using the existing token standard – they will be required for interacting with `ProgrammableNonFungible` assets. At the same time, the **new instructions will support all asset classes** so all interaction can happen via an unified interface regardless of the asset class.
```
Comment (samuelvanderwaal) [inline token-metadata/program/ProgrammableNFTGuide.md:47]: a new set of instruction**s**

Comment (samuelvanderwaal) [reply token-metadata/program/ProgrammableNFTGuide.md:47]: > It is important to note that current instructions will continue to work using the existing token standard – they will be required for interacting with `ProgrammableNonFungible` assets.

This sentence is confusing. Is `they` referring to the current instructions?

File: token-metadata/program/ProgrammableNFTGuide.md [hunk 2]
```diff
@@ -4,320 +4,262 @@
...
 Token Metadata instruction will be expanded to include:
 
 ```rust
 pub enum MetadataInstruction {
     ..,
-		// Mint a token with added metadata
+    // Create the metadata and associated accounts
+    Create(CreateArgs),
+    // Closes the accounts of an asset
+    Burn(BurnArgs),
```
Comment (samuelvanderwaal) [inline token-metadata/program/ProgrammableNFTGuide.md:57]: Looks like this list was ordered to be alphabetical, but if that's the case, `Burn` should come before `Create`.

File: token-metadata/program/ProgrammableNFTGuide.md [hunk 3]
```diff
@@ -4,320 +4,262 @@
...
+    // Updates the metadata of an asset
+    Update(UpdateArgs),
+    // Authorizes the use of a token
+    UseAsset(UseAssetArgs),
+    // Verifies creator/collection for an asset
+    Verify(VerifyArgs),
 }
 ```
 
 Each of these instruction will use versioned `*Args` structs to facilitate future updates, and in turn, not require additional instructions.
```
Comment (samuelvanderwaal) [inline token-metadata/program/ProgrammableNFTGuide.md:79]: Each of these instruction**s**

**Review by samuelvanderwaal — CHANGES_REQUESTED** [review]

**Review by lorisleiva — APPROVED** [review]
Looks really nice!

File: token-metadata/program/ProgrammableNFTGuide.md (hunk 3 above)
Comment (febo) [reply token-metadata/program/ProgrammableNFTGuide.md]: Done.

File: token-metadata/program/ProgrammableNFTGuide.md (hunk 2 above)
Comment (febo) [reply token-metadata/program/ProgrammableNFTGuide.md:55]: Done.

File: token-metadata/program/ProgrammableNFTGuide.md (hunk 1 above)
Comment (febo) [reply token-metadata/program/ProgrammableNFTGuide.md]: Tweak the sentence:

It is important to note that current instructions will continue to work using the existing token standards – the new instructions will be required for interacting with `ProgrammableNonFungible` assets.

Comment (samuelvanderwaal) [reply token-metadata/program/ProgrammableNFTGuide.md]: Perfect.

**Review by samuelvanderwaal — APPROVED** [review]

**Pull request merged**